doc/tools/docgram/orderedGrammar

```diff
@@ -1053,6 +1053,8 @@ search_item: [
 ]
 
 logical_kind: [
+| "Fixpoint"
+| "CoFixpoint"
 | [ thm_token | assumption_token ]
 | [ "Definition" | "Example" | "Context" | "Primitive" | "Symbol" ]
 | [ "Coercion" | "Instance" | "Scheme" | "Canonical" | "SubClass" ]
```

Comment:
editing this to be
```suggestion
| [ "Coercion" | "Instance" | "Scheme" | "Canonical" | "SubClass" | "Fixpoint" | "CoFixpoint" ]
```
(or whatever order you prefer) 
may work

Reply:
Hmm, didn't work. Maybe I did not do what was needed to please the gods. I would say it's not worth fighting for.

Reply:
@coqbot run full ci

Reply:
I pushed a commit

Reply:
@Villetaneuse The grammar documentation tool is described in `doc/tools/docgram/README.md`